modules/installation-osp-fixing-subnet.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// * installing/installing_openstack/installing-openstack-installer-user.adoc
+//
+//YOU MUST SET AN IFEVAL FOR EACH NEW MODULE
+
+[id="installation-osp-fixing-subnet_{context}"]
+= Setting a custom subnet for machines
+
+The IP range that the installation program chooses may not match the Neutron subnet that you create when installing {product-title} on your own infrastructure. If necessary, update the CIDR value for new machines by editing the installation configuration file.
+
+.Prerequisites
+
+* You have `install-config.yaml` as generated by the {product-title} installation program
+
+.Procedure
+
+. On a command line, browse to the directory that contains `install-config.yaml`.
+. From that directory, enter the command:
++
+----
+$ python -c '
+import yaml;
+path = "install-config.yaml";
+data = yaml.safe_load(open(path));
+data["networking"]["machineNetwork"][0]["cidr"] = "<subnet range>"; <1>
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+----
+<1> Insert a value that matches your intended Neutron subnet, e.g. `192.0.2.0/24`.
+
+[NOTE]
+You can also edit the file directly. In `install-config.yaml`, edit `networking.machineNetwork.network.cidr`.
```

Comment:
again, as of 4.2 it is:
neworking.machineCIDR

here is an example from a install config file: 
networking:
  clusterNetwork:
  \- cidr: 10.128.0.0/14
    hostPrefix: 23
  machineCIDR: 192.0.2.0/24
  networkType: OpenShiftSDN
  serviceNetwork:
  \- 172.30.0.0/16